hasjob/models/jobpost.py

```diff
@@ -228,7 +228,8 @@ def expiry_date(self):
     def after_expiry_date(self):
         return self.expiry_date + timedelta(days=1)
 
-    state.add_conditional_state('OLD', state.LISTED, lambda proposal: proposal.datetime < datetime.utcnow() - agelimit, label=('old', __("Old")))
```

Comment:
This is correct, but it's a jobpost, not a proposal. OLD is a conditional state for LISTED.

Reply:
@jace right. typo. I removed it thinking OLD should not be a state. but yeah, it is only a conditional state for LISTED. so it makes sense. I'll revert it back.